lib/chef-dk/skeletons/code_generator/recipes/build_cookbook.rb

```diff
@@ -122,7 +122,14 @@
     command("git add .delivery/config.json")
     cwd delivery_project_dir
 
-    only_if "git status --porcelain |grep \".\""
+    only_if (
+	    case [:os_family]
```

Comment:
There were tabs in notepad++, corrected in next commit, having trouble with the rebase to edit this commit comment